ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.9.yaml

```diff
@@ -155,34 +155,34 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ocm-ci-image-mirror
-- as: release-280-operator-image-mirror
+- as: release-290-operator-image-mirror
   postsubmit: true
   steps:
     dependencies:
       SOURCE_IMAGE_REF: multicluster-global-hub-operator
     env:
       IMAGE_REPO: multicluster-global-hub-operator
-      IMAGE_TAG: v0.7.0
+      IMAGE_TAG: v0.8.0
```

Comment:
use 1.0.0 instead as we discussed this morning.

Reply:
Done!